src/register_types.cpp

```diff
@@ -55,9 +54,13 @@ void initialize_module(ModuleInitializationLevel p_level) {
 }
 
 void uninitialize_module(ModuleInitializationLevel p_level) {
-	if (p_level == MODULE_INITIALIZATION_LEVEL_CORE) {
-		memdelete(SentrySDK::get_singleton());
-		delete SentryOptions::get_singleton();
+	if (p_level == MODULE_INITIALIZATION_LEVEL_SCENE) {
```

Comment:
Is this the latest step? Could we uninstall even later?

Reply:
I'll see if I can push it to earlier/later stages. It's been problematic with LEVEL_CORE so far.

Reply:
Opened https://github.com/getsentry/sentry-godot/issues/39